bin/setup

```diff
@@ -54,6 +54,17 @@ chdir APP_ROOT do
     end
   end
 
+  step "Installing jq, a JSON CLI used by our release script" do
+    # use --version flag because running `chromedriver` will start
+    # chromedriver, which stops the script indefinitely if already installed.
+    if cli_installed?("jq")
```

Comment:
woo

Reply:
tempted to use a Brewfile for dependencies like this